packages/core/addon/components/table-renderer-vertical-collection.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ * {{#table-renderer-vertical-collection
+ *   tableData=tableData
+ *   request=request
+ *   columns=columns
+ *   occlusion=occlusion
+ *   estimateHeight=estimateHeight
+ *   bufferSize=bufferSize
+ *   isEditingMode=isEditingMode
+ *   cellRendererPrefix=cellRendererPrefix
+ *   headerClicked=(action 'headerClicked')
+ *   updateColumnOrder=(action 'updateColumnOrder')
+ *   updateColumnDisplayName=(action 'updateColumnDisplayName')
+ *   onUpdateReport=(action onUpdateReport)
+ * }}
+ */
+import Component from '@ember/component';
+import layout from '../templates/components/table-renderer-vertical-collection';
+import { computed, get, set } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { run } from '@ember/runloop';
+
+/**
+ * @constant {String} SCROLL_EVENT - Scroll event name
+ */
+const SCROLL_EVENT = 'scroll';
+
+/**
+ * @constant {String} RESIZE_EVENT - Resize event name
+ */
+const RESIZE_EVENT = 'didResize';
+
+export default Component.extend({
+  layout,
+
+  classNames: ['table-widget-container'],
+
+  /**
+   * @property {Service} resize - Resize service to respond to view resize events
+   */
+  resize: service(),
+
+  /**
+   * @property {String} tableWrapperClass - class of table wrapper div
+   */
+  tableWrapperClass: 'table-widget__table-wrapper',
+
+  /**
+   * @property {Object} tableWrapperDomElement - table wrapper div DOM element
+   */
+  tableWrapperDomElement: computed('tableWrapperClass', function() {
```

Comment:
`tableWrapperClass` is private correct? Do we expected it to update? If not, can we remove the listener?

Reply:
We don't, by listener you mean the `'tableWrapperClass'` part?